devito/propagator.py

```diff
@@ -308,7 +308,11 @@ def generate_loops(self, loop_body):
         omp_master = [cgen.Pragma("omp master")] if self.compiler.openmp else []
         omp_single = [cgen.Pragma("omp single")] if self.compiler.openmp else []
         omp_parallel = [cgen.Pragma("omp parallel")] if self.compiler.openmp else []
-        omp_for = [cgen.Pragma("omp for schedule(static)")] if self.compiler.openmp else []
+        reduction_list = ["kernel", "loop_body"] if self.compiler.openmp else []
```

Comment:
Well as we discussed just now, the other parts of the profiling code are also hard coding the these stuff, so I think this is a more general issue that deserves another PR.